rest_framework/permissions.py

```diff
@@ -109,8 +110,6 @@ def get_required_permissions(self, method, model_cls):
     def has_permission(self, request, view):
         try:
             queryset = view.get_queryset()
-        except AttributeError:
-            queryset = getattr(view, 'queryset', None)
```

Comment:
Should we just be using exactly the same style as here?... https://github.com/tomchristie/django-rest-framework/pull/2863/files#diff-d5f4b02a7b88b58fe3990de607a06d8fR110